database/cockroachdb/cockroachdb.go

```diff
@@ -85,11 +86,12 @@ func WithInstance(instance *sql.DB, config *Config) (database.Driver, error) {
 		config: config,
 	}
 
-	if err := px.ensureVersionTable(); err != nil {
+	// ensureVersionTable is a locking operation, so we need to ensureLockTable before we ensureVersionTable.
```

Comment:
Thanks for finding and fixing this issue!
The comment is useful.

It's unfortunate that lock table creation will have issues when using multiple instances of migrate and cockroachdb and there's nothing we can do about it....

Reply:
Yeah, I guess it would be possible to lock it on the Driver-level, but that only helps for that specific Driver instance, doesn't help if there are multiple Drivers running on the same database. :(